docs/encyclopedia/workers.mdx

```diff
@@ -190,6 +191,30 @@ Once an Activity Task finishes execution, the Worker responds to the Temporal Se
 - ActivityTaskTerminated
 - ActivityTaskTimedOut
 
+## What is a Nexus Operation Task? {#nexus-operation-task}
+
+A Nexus Operation Task provides the context needed to proceed with an Nexus Operation Task Execution.
+Nexus Operation Tasks largely represent the Nexus Operation Scheduled Event, containing the data needed to execute a Nexus Operation handler.
+
+## What is a Nexus Operation Task Execution? {#nexus-operation-task-execution}
+
+An Nexus Operation Task Execution occurs when a Worker uses the context provided from the Nexus Operation Task and executes the Nexus Operation handler (also known as the Nexus Operation handler function).
+
+The NexusOperationScheduled Event corresponds to when the Temporal Service puts the Nexus Operation Task into the Task Queue.
```

Comment:
```suggestion
The NexusOperationScheduled Event corresponds to when the Temporal Service puts the Nexus Task to start a Nexus Operation into the Task Queue.
```

Reply:
The NexusOperationScheduled Event corresponds to when the Temporal Service records the Workflow's intent to schedule an operation.